tests/unit/select_connection_ioloop_tests.py

```diff
@@ -10,22 +10,33 @@
 
 import logging
 try:
+    #pylint: disable=F0401
```

Comment:
Thanks @shinji-s. I stand corrected. I also discovered that with my pylint version 1.4.4, pylint is smart about the ImportError exception. In the following code block, it only complains F0401 about `import halbhalbhalb`, but doesn't complain about `import blahblahblah`

```
try:
  import blahblahblah
except ImportError:
  import datetime

import halbhalbhalb
```